### PR TITLE
Add a few faqs for development.

### PR DESCRIPTION
**Pull request by jomeier**
Hi,

I collected a few faqs for people interested in development. I'm always searching for certain commands. I'm sure there is much more which will follow.

Greetings,

Josef

### CODE REVIEW

File: FAQ.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+# Frequently Asked Questions
+
+## Where can I find upgrades?
+https://origin-release.svc.ci.openshift.org/
+
+Each Preview/Beta version is marked with an asterix *
```
Comment (vrutkovs) [inline FAQ.md:6]: asterisk means the build won't be pruned. We may want to keep arbitrary nightlies, regardless of Beta status.

I don't think its worth mentioning, its release-controller detail

File: FAQ.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+Find a version where a tested upgrade path is available from your version for on 
+
+https://origin-release.svc.ci.openshift.org
+
+Upgrade options:
+
+**Preferred** ways:
+* Web Console: Home -> Overview -> Tab: Cluster, Card: Overview -> View settings -> Update Status
+
+  Currently (as of 2020-04-10) this doesn't work because the nightly builds are not signed. Will be resolved soon !
```
Comment (vrutkovs) [inline FAQ.md:22]: Lets mention a tracking bug for that - https://github.com/openshift/okd/issues/138

File: FAQ.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+Cloud provider secrets, ... will not be included.
+
+Upload it to a file hoster and send the link to the developers (Slack channel, ...)
+
+# External tips for OKD 4
+
+* slack:
+
+  Workspace: Kubernetes, Channel: #openshift-dev
```
Comment (vrutkovs) [inline FAQ.md:109]: I'd prefer to have user conversations in `#openshift-user` and have `-dev` for developer communication. Lets mention both

**Comment by vrutkovs** [comment]
Thanks!

File: FAQ.md (hunk 3 above)
Comment (jomeier) [reply FAQ.md]: Done

File: FAQ.md (hunk 2 above)
Comment (jomeier) [reply FAQ.md]: Done

File: FAQ.md (hunk 1 above)
Comment (jomeier) [reply FAQ.md]: I made a hint out of this comment. I stumbled of that. If it's resolved that nightlies won't be deleted I think this comment can be removed. It's on you to decide that.

**Pull request merged**